qlib/config.py

```diff
@@ -172,6 +172,7 @@ def register_from_C(self, config, skip_register=True):
             }
         },
         "loggers": {"qlib": {"level": logging.DEBUG, "handlers": ["console"]}},
+        "disable_existing_loggers": False,
```

Comment:
Could you please add comments for this line?

Reply:
```
Add the following config line to logging_config to prevent disabling the existing logger.
Without this line, loggers from other code will be disabled.
```
The content in your PR looks good.

Reply:
Sure, added.